src/main/java/org/springframework/integration/kafka/core/KafkaMessage.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2014 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package org.springframework.integration.kafka.core;
+
+import kafka.message.Message;
+
+/**
+ * Wrapper around a {@link KafkaMessage}, grouping the original {@link Message} content and metadata.
```

Comment:
`Wrapper around a {@link Message kafka message} and its {@link KafkaMessageMetadata metadata}.`